apps/condo/domains/common/utils/timezone.js

```diff
@@ -13,6 +15,15 @@ const normalizeTimeZone = (timeZone) => {
     }
 }
 
+const dayjsWithTimezone = ({ date, timezone }) => {
+    try {
+        return dayjs(date).tz(timezone)
+    } catch (e) {
```

Comment:
And also are we need this fallback? Maybe just handle this on front-end part?
It can cause some unpredicted behaviours potentially.

1. Client A somehow has correct time zone in one tab and incorrect in another
2. Client will use this util at almost the same time, but he'll get few hours diff between 2 objects...